guides/introduction/Embedded Schemas.md

```diff
@@ -1,89 +1,84 @@
 
 ```elixir
 defmodule User do
   use Ecto.Schema
 
   schema "users" do
-    field :is_active, :boolean
+    field :full_name, :string
     field :email, :string
+    field :avatar_url, :string
     field :confirmed_at, :naive_datetime
 
-    embeds_one "profile" do
-      field :age, :integer
-      field :favorite_color, Ecto.Enum, values: [:red, :green, :blue, :pink, :black, :orange]
-      field :avatar_url, :string
+    embeds_one :profile do
+      field :online, :boolean
+      field :dark_mode, :boolean
+      field :visibility, Ecto.Enum, values: [:public, :private, :friends_only]
     end
 
     timestamps()
   end
 end
 ```
 
-Let's kick things off with a tutorial to explain how to recreate this use case: building a solution for user profiles in an example app.
-
-### Writing the schema
-
-The first step is to write the structure of the data we're storing. In the case of our profile, we'd like to store the user's age, favorite color, and a profile picture.
+### Embeds
 
-To begin writing this embedded schema, we must first think about what structure we want. Do we want to store an array of structures (using `embeds_many`) or just one (using `embeds_one`)?
-
-In this case, every user should have only one profile associated to them, so we'll begin by writing like any other Ecto schema:
+There are two ways to represent embedded data within a schema, `embeds_many`, which creates a list of embeds, and `embeds_one`, which creates only a single instance of the embed. Your choice here affects the behavior of embed-specific functions like `Ecto.Repo.put_embed/4` and `Ecto.Repo.cast_embed/4`, so choose whichever is most appropriate to your use case. In our example we are going to use `embeds_one` since users will only ever have one profile associated with them.
 
 ```elixir
 defmodule User do
   use Ecto.Schema
 
   schema "users" do
-    field :is_active, :boolean
+    field :full_name, :string
     field :email, :string
+    field :avatar_url, :string
     field :confirmed_at, :naive_datetime
 
     embeds_one :profile do
-      field :age, :integer
-      field :favorite_color, Ecto.Enum, values: [:red, :green, :blue, :pink, :black, :orange]
-      field :avatar_url, :string
+      field :online, :boolean
+      field :dark_mode, :boolean
+      field :visibility, Ecto.Enum, values: [:public, :private, :friends_only]
     end
 
     timestamps()
   end
 end
 ```
 
-We can, however, clean this up a little. You can separate the profile to a distinct module and keep the `User` module tidy:
+### Extracting the embeds
+
+While the above User schema is simple and sufficient, we might want to work independently with the embedded profile struct. For example, if there were a lot of functions devoted solely to manipulating the profile data. In such scenarios, it is recommended to extract the embedded struct into its own schema using the `embedded_schema` function.
```

Comment:
> embedded profile struct if, for example, there were...

The separate `For example,...` sentence reads a little funky.

Reply:
reworked the paragraph based on the feedback <3